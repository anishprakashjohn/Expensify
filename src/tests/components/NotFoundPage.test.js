import React from 'react';
import {shallow} from 'enzyme';
import NotFoundPage from '../../component/NotFoundPage';

test('should render NotFoundpage', () => {
    const wrapper = shallow(<NotFoundPage />)
    expect(wrapper).toMatchSnapshot();
})