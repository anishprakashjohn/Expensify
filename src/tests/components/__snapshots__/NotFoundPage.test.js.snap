// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`should render NotFoundpage 1`] = `
<p>
  404! - 
  <Link
    replace={false}
    to="/edit"
  >
     Go Home 
  </Link>
</p>
`;
